### ISSUE TITLE
SwitchAccount gets stuck

### ISSUE THREAD

**Issue by nitishpatel** [ChainSafe/dappeteer#94]
**Describe the bug**
In my use case, I need to switch to the second account, however, the account is not created itself so, the `switchAccount` method just freezes 

**To Reproduce**
`await metamask.switchAccount(2);`

**Logs**

```
const timeoutError = new Errors_js_1.TimeoutError(`waiting for ${options.title} failed: timeout ${options.timeout}ms exceeded`);
                                 ^

TimeoutError: waiting for selector `.account-menu__accounts > div:nth-child(2)` failed: timeout 30000ms exceeded
    at new WaitTask (D:\courses\automation\tokenization-automation\node_modules\puppeteer\lib\cjs\puppeteer\common\DOMWorld.js:526:34)
    at DOMWorld.waitForSelectorInPage (D:\courses\automation\tokenization-automation\node_modules\puppeteer\lib\cjs\puppeteer\common\DOMWorld.js:430:26)
    at Object.internalHandler.waitFor (D:\courses\automation\tokenization-automation\node_modules\puppeteer\lib\cjs\puppeteer\common\QueryHandler.js:31:77)
    at DOMWorld.waitForSelector (D:\courses\automation\tokenization-automation\node_modules\puppeteer\lib\cjs\puppeteer\common\DOMWorld.js:321:29)
    at Frame.waitForSelector (D:\courses\automation\tokenization-automation\node_modules\puppeteer\lib\cjs\puppeteer\common\FrameManager.js:925:51)
    at Page.waitForSelector (D:\courses\automation\tokenization-automation\node_modules\puppeteer\lib\cjs\puppeteer\common\Page.js:2359:33)
    at D:\courses\automation\tokenization-automation\node_modules\@chainsafe\dappeteer\dist\metamask\switchAccount.js:18:32
    at Generator.next (<anonymous>)
    at fulfilled (D:\courses\automation\tokenization-automation\node_modules\@chainsafe\dappeteer\dist\metamask\switchAccount.js:5:58)
    at runMicrotasks (<anonymous>)
```

**Expected behavior**
To Switch the account

**Screenshots**
![image](https://user-images.githubusercontent.com/39486276/154244376-eaaad531-7001-4c3e-b945-f0ec162ef6b6.png)

**System:**
- OS: Windows 
- OS version 11
- NodeJs version 16.10.0
- dAppeteer version 2.4.0
- testing framework dappeteer
- testing framework version NA 2.4.0

**Comment by mojinet** [comment]
Hi, i have the same problem and i fix.

to fix : modify metamask/importPk.js => line 22 change button.btn-secondary to button.btn-primary

**Comment by nitishpatel** [comment]
> Hi, i have the same problem and i fix.
> 
> to fix : modify metamask/importPk.js => line 22 change button.btn-secondary to button.btn-primary

Can you create a PR?

**Comment by mojinet** [comment]
it's done : https://github.com/ChainSafe/dappeteer/commit/0fc80199cd1511835a8240400517f75fd48aae6e

**Comment by nitishpatel** [comment]
is it related to switchAccount tho? I think we need some method like addAccount which creates account and then use swtichAccount to switch the account

**Comment by iamkiko** [comment]
I assumed importPK() would allow to switch accounts and therefore have a different ethereum address.

I am also noticing that the account number is the same as your screenshot (ending in f292).

Have you managed to change the account altogether?

**Comment by nitishpatel** [comment]
Right but in order to use importPK() I need to have exported Pvt keys for all the accounts under a passphrase, lets's say in my case I  am using around 7-8 accounts while testing the Dapp, and maintaining Pvt keys for all via environment variables would be a challenge.

**Comment by nitishpatel** [comment]
Any updates on this?

**Comment by mpetrunic** [comment]
should be fixed in 3.0.0-rc.0
